docs/lago-tutorial/README.rst

```diff
@@ -0,0 +1,242 @@
+Lago-Tutorial
+=============
+
+This repository contains the default files for creating Lago
+environment.
+
+In this tutorial you will learn how to set up a custom virtual
+environment with Lago.
+The virtual environment will be used as an infrastructure for your
+application, so you can run tests
+in order to check the functionality of your application, without the
+overhead of setting up an infrastructure.
+
+This tutorial will be “hands on”, and will take you through the process
+of creating, running and deploying the environment.
+
+Lago environments are flexible and can be used for all different kinds
+of tasks. For keeping things simple I will focus on one scenario, using only basic commands of Lago,
+although you will get the ability to create your own custom environment.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Install Lago - follow
+`this <../README.html#installation>`__ tutorial
+for more information
+
+The scenario
```

Comment:
this should go with the jenkins scenario, here only generic stuff should be explained